### ISSUE TITLE
Percentile Aggregate on Groupby

### ISSUE THREAD

**Issue by jdbodyfelt** [vaexio/vaex#1998]
**Description**
As  user, I would like to run a `groupby().describe()`, but this is not supported in Vaex yet. I can use the `vaex.agg` family to build a similar structure to `describe()` for use in a `groupby` aggregation:
`aggs = {
        "count": vaex.agg.count(col),
        "mean": vaex.agg.mean(col),
        "std": vaex.agg.std(col),
        "min": vaex.agg.min(col),
        "max": vaex.agg.max(col),
    }
    stats = df[[*groups, col]].groupby(by=groups, progress=True).agg(aggs)`
    
Is it possible to the vaex.agg family to either:
a) include percentiles, e.g. something like `vaex.agg.percentile(col, 0.30)` on a groupby? 
b) have `describe()` work in a `groupby()` aggregation? 

Thank you! 

**Is your feature request related to a problem? **
There's a few other open issues that may be related (e.g. `groupby().apply()`), but this is a general stats (e.g. `np` supported) aggregation, so the issues of compute map complexities may not be a problem...

**Comment by JovanVeljanoski** [comment]
Hi, 

Thanks for your submission. I think:
a) That is definitely possible, and it is on our radar, but I can't comment on how quickly it can happen (if not part of some funded work).  PRs are very much welcome of course!

b) That should be possible, I will see if I can implement it next chance I have. Maybe you (or anyone interested) can try to add such a method to `groupby.GroupBy`. It is basically a short-cut as you very rightly pointed out.

**Comment by jdbodyfelt** [comment]
Zdravo Jovan, 

    Thanks for adding this as a feature in the planning. I'd propose an interface copy of pandas' `describe()`, including that `percentile=` option. I was digging into `agg.py` - perhaps a `AggregatorDescriptorQuantile(AggregatorDescriptorMulti)` can do the trick...

**Comment by JovanVeljanoski** [comment]
Yeah, I would like to have that functionality also (adding an optional argument to `df.describe()`). 
Do you want to try and open a PR for this?

As for an aggregator, you can try that also :). Not sure if that would require touching the c++ layer, maybe @maartenbreddels can give more info if you are interested in giving it a go!

**Comment by jdbodyfelt** [comment]
Ah, I didn't think to go into `groupby.Groupby` - that's a possibility. I'm happy to write a percentile method for that class, but can't promise a quick implementation. I'll clone and raise a PR on that later this week...
